elastic/kibana
# [Infrastructure UI] Fix Hosts View date picker for some preset values

## 📓 Summary

Closes #151150 

The issue was caused by a missing rounding up when the same string represented dates.

## 🧪 Testing

- Navigates to Hosts View page.
- Set date-picker to `Today`.
- Verify data are correctly fetched.
Repeat the same setting the picker to `This week`

https://user-images.githubusercontent.com/34506779/221857418-dbc53fe6-6eca-4645-93cd-b939231e76b0.mov
